contrib/dash/exsrvmonit/exsrvmonit.py

```diff
@@ -26,8 +26,7 @@
 MIN_PROTO_VERSION = '1.4'
 NUM_RECENT_DATA = 1440
 SERVERS_LIST = [
-    'electrum.dash.siampm.com:50002',
-    'drk.p2pay.com:50002',
```

Comment:
The drk.p2pay.com server appears to be responding to network calls, so please let me know why you removed it. Does it not support the latest protocol changes or something else?

Reply:
drk.p2pay.com is responding, but it looks like it chainforked or stuck at 1944770 block

<img width="657" alt="image" src="https://github.com/Bertrand256/electrum-dash/assets/17009187/6cdf3012-f299-48b0-918d-7349531831b1">

People fails to use dash electrum when this server is chosen